docs/notebooks/Common_Gotchas_in_JAX.ipynb

```diff
@@ -1627,7 +1599,9 @@
         "id": "0pcn2LeS-03b"
       },
       "source": [
-        "JAX and XLA offer the very general N-dimensional __conv_general_dilated__ function, but it's not very obvious how to use it.  We'll give some examples of the common use-cases.  There are also the convenience functions `lax.conv` and `lax.conv_general_padding` for the most common kinds of convolutions.\n",
+        "JAX and XLA offer the very general N-dimensional __conv_general_dilated__ function, but it's not very obvious how to use it.  We'll give some examples of the common use-cases.\n",
+        "\n",
+        "For the most common kinds of convolutions, see also the convenience functions lax.conv and lax.conv_general_padding, as well as jax.numpy.convolve and jax.scipy.signal.convolve/jax.scipy.signal.convolve2d for an interface similar to that of the numpy and scipy packages.\n",
```

Comment:
This block contains the operative change.

Reply:
I wonder if there are best practices for editing these files to avoid incidental changes?

Reply:
I think different notebook editors have different conventions regarding order of fields within the JSON. Options to get around this would be to edit the text directly, or to have all contributors use the same version of Jupyter. Neither is ideal, unfortunately.